### PR TITLE
sign up layout front-end 

### PR DESCRIPTION
**Pull request by SaharFroukh**
### sign up layout front-end #27 

Description:
Layout of sign up page using Material UI,  to open this page you have to add /user/Signup to the url.

### CODE REVIEW

File: client/src/Components/Layout/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+              color="primary"
+              variant="contained"
+            >
+              Sign up
+            </Button>
+            <Typography variant="h7" color="primary">
+              {"Have an account?  "}
+              <Link href="#" color="#FB991C">
+                login
+              </Link>
```
Comment (YakoobHammouri) [inline client/src/Components/Layout/Signup.jsx]: this link must using the Link from  react-router-dom 

`import { Link } from 'react-router-dom';`

```
<Link to={`/user/login}`}>
          Login
</Link>
```

**Review by YakoobHammouri — CHANGES_REQUESTED** [review]

File: client/src/Components/Layout/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            </Typography>
+          </Grid>
+        </form>
+      </Box>
+    );
+  }
+}
+
+export default SignUP;
```
Comment (sara219) [inline client/src/Components/Layout/Signup.jsx]: Please follow the naming guide ( PascalCase ) to naming your components

**Review by sara219 — CHANGES_REQUESTED** [review]
That what I see when I test your work, fix it, please 

Screenshot from 2020-04-26 13-47-09

**Comment by sara219** [comment]
Can you add the endpoint into `UserLayout`!

File: client/src/Components/Layout/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import React, { Component } from "react";
```
Comment (sara219) [inline client/src/Components/Layout/Signup.jsx]: why we have two signup component!

**Review by sara219 — CHANGES_REQUESTED** [review]

File: client/src/Components/Layout/SignUp.jsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  Grid,
+  TextField,
+  Button,
+  Link,
+  Typography,
+} from "@material-ui/core";
+import { Person, AlternateEmail, Phone, Lock } from "@material-ui/icons";
+import { withStyles } from "@material-ui/core/styles";
+
+class SignUP extends Component {
```
Comment (sara219) [inline client/src/Components/Layout/SignUp.jsx:14]: here the component name is SignUP
but when you export it you write signUp   `export default signUp;`
its should be the SAME

**Review by sara219 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**